spec/models/booking_spec.rb

```diff
@@ -30,6 +30,58 @@
 
       expect(Booking.last_week.count).to eq(2)
     end
+
+    it 'should return bookings with bondable charges' do
```

Comment:
Instead of `describe 'scopes'`, could we break the describe block down by the scope that's being tested? It might make it easier to skim the test file.

e.g.
```
describe '.bondable' do
  it 'blah blah blah'
end

describe '.active' do
  it 'blah blah blah'
end
```